Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIWebNavigationCocoa.mm

```diff
@@ -53,8 +100,9 @@
         return;
     }
 
-    // FIXME: rdar://102820594 - Implement this.
-    completionHandler({ });
+    [webView _frames:makeBlockPtr([this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler), tab = WTFMove(tab), frameIdentifier = WTFMove(frameIdentifier)] (_WKFrameTreeNode *mainFrame) mutable {
```

Comment:
```
makeBlockPtr([this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler), tab, frameIdentifier]
```

`tab` and `frameIdentifier` can just be copied since it is `RefPtr` and basically a `uint64_t`.